domains/certificates/RegistryConnector/Worker/RetryOptions.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace RegistryConnector.Worker;
+
+public class RetryOptions
+{
+    public const string Retry = nameof(Retry);
+
+    [Required]
+    public int DefaultFirstLevelRetryCount { get; set; }
+
+    [Required]
+    public int DefaultSecondLevelRetryCount { get; set; }
```

Comment:
I expect this has something to do with the upper limit.
We have an AC saying:
When reaching the upper limit, it must be possible to monitor that there is a certificate that the issuer have given up on.

Do we do that?

Reply:
Only through logging. When talking to Paul he said that metrics was a different story